main.cpp

```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
         c) pass the local variables to ignoreUnused() as you did in 2b)
     see main() for an example of this.
  
-10) click the [run] button.  Clear up any errors or warnings as best you can.
- */
+10) click the [run] buttn.  Clear up any errors or warnings as best you can.
 
+*/
 
 //2)
 void variableDeclarations()
 {
     //example:
-    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
+    int myHeight = 72;    
+    int howManyTrees = 8;
+    int moneyEarned = 0;
+
+    float temperature = 32.0f; 
+    float rotations = 2.375f;
+    float exactChange = 12.86f;
+
+    bool doorIsOpen = true;
+    bool makeASound = false;
+    bool activateSwitch = true;
+
+    double numberOfTracks = 4; 
+    double inputsAllowed = 24;
+    double eqModesShown = 6;
```

Comment:
Don't initialize doubles with integers.
write doubles correctly. 
`4.0` is a double.  `4` is an integer
- [ ] revise